routes/employee/index.js

```diff
@@ -6,9 +6,9 @@ import authenticate from "../../middlewares/authenticate.js";
 import upload from "../../middlewares/multer.js";
 
 const router = express.Router();
-router.get("/", authenticate, controllers.getAll);
+router.get("/", controllers.getAll);
```

Comment:
Why employee Update in Payroll Pagination?

Reply:
Sir in the previous code there was not user Authentication for the Get By Id i just added the authentication.